doc/manual/migrating_from_eeglab.rst

```diff
@@ -0,0 +1,68 @@
+Caveats coming from EEGLAB
+==========================
+
+EEGLAB vs. MNE: Pros and Cons
+-----------------------------
+
+Major differences: MNE-Python ...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* is **Python based**. Python is a fully featured, general programming language. It is easy to learn, easy to read (and thus share, criticize and reproduce code with). It is also free - unlike MATLAB, which you need for EEGLAB, and the MATLAB toolboxes. For example, with MNE, you will never run out of/lack parallel processing toolboxes, and thus MNE more readily uses multiple cores (usually with the `n_jobs` keyword).
+* was born as a package for **MEG analysis**. While *MNE has gained strong EEG support*, it in many instances takes a more extensive approach. For example, MEG data is usually multimodal (including magnetometers, gradiometers and EEG), and thus MNE requires you to constantly choose what channel type you want to operate with. MNE also has much more integrated support for source-level/source localisation than EEGLAB.
+* is **Object oriented**. Typical user-exposed functions don't operate on raw data matrices, but on `Raw`, `Epochs`, `Evoked`, ... objects. Many object methods operate in-place. For example, when in MNE, you do EEG = pop_eegfiltnew(EEG, ...), in MNE, you do raw.filter(...). In EEGLAB, you do pop_plot_topo(EEG, ...). In MNE, you do evoked.plot_topomap(...). In EEGLAB, you do pop_saveset(EEG, ...). In MNE, you do `epochs.save(...)`. `raw.plot()` is very different from `evoked.plot()`. In-place modification can usually be turned off with the `copy=True` setting.
+* does **not feature a GUI**. MNE always requires explicit code. This can be frustrating for beginners, but in the long run, it supports good scientific practices - better reproducible, more scalable analyses. Generally speaking, there is some trade-off between the lack of GUI in MNE and the Python base; while GUIs are more accessible to newcomers, Python more easily allows experienced users to create custom and advanced analysis pipelines and techniques.
```

Comment:
maybe you can point out that even though we don't support GUI, we do allow interactive exploration through our plotting routines, c.f., epochs.plot() and interactive projs etc.